dev/release/verify-release-candidate.sh

```diff
@@ -904,6 +904,7 @@ test_integration() {
   maybe_setup_virtualenv
 
   pip install -e dev/archery[integration]
+  pip install jpype1 || :
```

Comment:
Hmm, what does the trailing part (`|| :`) do?

Reply:
`||` is the or operator and `:` is the null operator which just returns a successful exit code. My idea was at least do not fail if jpype cannot be installed.  As I recall reading that it might not be available on some systems.

Reply:
Why not try without this first?

Reply:
By null operator I mean it does nothing :)
https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Bourne-Shell-Builtins

Reply:
Well, regardless, let's just first try without it?

Reply:
my concern was not with the CI jobs but that this is the verification script and could potentially be executed on other systems but I am ok removing it and deal with any potential issue if it arises.